client/src/elm/Backend/Endpoints.elm

```diff
@@ -1207,3 +1210,20 @@ ncdHbA1cTestEndpoint : ReadWriteEndPoint Error NCDHbA1cTestId NCDHbA1cTest NCDHb
 ncdHbA1cTestEndpoint =
     swEndpoint "nodes/ncd_hba1c_test" decodeNCDHbA1cTest
         |> withValueEncoder (object << encodeNCDHbA1cTest)
+
+
+resilienceSurveyEndpoint : ReadWriteEndPoint Error ResilienceSurveyId ResilienceSurvey ResilienceSurvey (Maybe NurseId)
+resilienceSurveyEndpoint =
+    swEndpoint "nodes/resilience_survey" decodeResilienceSurvey
+        |> withValueEncoder (object << encodeResilienceSurvey)
+        |> withParamsEncoder encodeResilienceSurveyParams
+
+
+encodeResilienceSurveyParams : Maybe NurseId -> List ( String, String )
+encodeResilienceSurveyParams params =
+    case params of
+        Just id ->
```

Comment:
`Just uuid`

Reply:
We always use `id` for IDs, no matter if it's Int, String or UUID.